samples/attested_tls/client/enc/CMakeLists.txt

```diff
@@ -5,8 +5,10 @@
 add_custom_command(
   OUTPUT tls_client_t.h tls_client_t.c tls_client_args.h
   DEPENDS ${CMAKE_SOURCE_DIR}/client/tls_client.edl
-  COMMAND openenclave::oeedger8r --trusted
-          ${CMAKE_SOURCE_DIR}/client/tls_client.edl)
+  COMMAND
+    openenclave::oeedger8r --trusted ${CMAKE_SOURCE_DIR}/client/tls_client.edl
+    --search-path ${OE_INCLUDEDIR} --search-path
```

Comment:
NTH: Can we avoid requiring the user to specify `${OE_INCLUDEDIR}/openenclave/edl/sgx`?
Intel SDK aseems to require specifying only `$(SGX_SDK)/include`:
https://github.com/intel/linux-sgx/blob/fdc9b33ee173a5e7743f20a2a9cb1dd8db114dca/SampleCode/Switchless/Makefile#L199
```make
	@cd App && $(SGX_EDGER8R) --untrusted ../Enclave/Enclave.edl --search-path ../Enclave --search-path $(SGX_SDK)/include
```

Reply:
See below comment.

Intel SDK can do that because they only support SGX